src/hooks/use-toast.test.ts

```
@@ -0,0 +1,262 @@
+      result.current.toasts.forEach((toast) => {
+        result.current.dismiss(toast.id);
+      });
+    });
+  });
+
+  beforeEach(() => {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    jest.useRealTimers();
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)